corehq/ex-submodules/phonelog/migrations/0008_devicelog_varchar_index.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-06-01 18:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import HqRunSQL
+from corehq.util.django_migrations import add_if_not_exists
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phonelog', '0007_devicelog_indexes'),
+    ]
+
+    operations = [
+        HqRunSQL(
+            add_if_not_exists(
```

Comment:
I don't think i can use `AlterIndexIfNotExists` since that relies on django to create the sql and it will not create the right sql.